lib/appsignal.ex

```diff
@@ -115,10 +115,11 @@ defmodule Appsignal do
   When there is no current transaction, this call starts one.
 
   """
-  def send_error(reason, message \\ "", stack \\ nil, metadata \\ %{}, conn \\ nil) do
+  def send_error(reason, message \\ "", stack \\ nil, metadata \\ %{}, conn \\ nil, fun \\ fn(t) -> t end) do
```

Comment:
Docs yes!

We need to make a page like this one for Elixir: http://docs.appsignal.com/ruby/instrumentation/exception-handling.html